### PR TITLE
Feature/gitlab prs

### PR DESCRIPTION
**Pull request by ppezaris**

### CODE REVIEW

File: shared/agent/src/@types/graphql.d.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+declare module "*.graphql" {
+	import { DocumentNode } from "graphql";
+
+	const value: DocumentNode;
```
Comment (ppezaris) [inline shared/agent/src/@types/graphql.d.ts:4]: OMG so ugly.

Comment (ppezaris) [reply shared/agent/src/@types/graphql.d.ts:4]: yes, yes it is.

**Comment by ppezaris** [comment]
test

**Comment by orellabac** [comment]
It his the PR to enable Gitlab Selfhosted MR reviews in codestream? I am really interested in this feature

**Comment by bcanzanella** [comment]
> 
> 
> It his the PR to enable Gitlab Selfhosted MR reviews in codestream? I am really interested in this feature
hey @orellabac it is!

**Pull request merged**